### PR TITLE
Moving build tool to shippable

### PR DESCRIPTION
**Pull request by notABluesSinger**
#### What are the links to relevant tickets?
https://hxshortbreaks.atlassian.net/browse/WEB-4809
#### What does this PR do?
Move the build tool to shippable
#### What functional/unit tests does this PR have?
Those that were left from the last developers work, but we now have full reporting on them.
#### How should a developer review this?
Reading.
#### Any background context you want to provide?
https://hxshortbreaks.atlassian.net/browse/FEA-442
#### Screenshots (if appropriate)
**Coverage Before**
Coverage
**Coverage After**
Coverage

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [ ] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @kevinhodges
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2 @lukefrake
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -35,11 +27,7 @@ module.exports = function( grunt ) {
 	} );
 
 	// load tasks from the specified grunt plugins...
-	grunt.loadNpmTasks( 'grunt-mocha-test' );
 	grunt.loadNpmTasks( 'grunt-contrib-jshint' );
 	grunt.loadNpmTasks( 'grunt-jscs' );
 
-	// register task alias'
```
Comment (lukefrake) [inline Gruntfile.js]: Can we make sure the readme is updated to reflect this please

File: package.json [hunk 2]
```diff
@@ -17,24 +17,25 @@
     "node": ">= 0.10.0"
   },
   "scripts": {
-    "test": "grunt test && istanbul cover _mocha -- test/ -R spec"
+    "test": "grunt jshint && grunt jscs && istanbul cover _mocha -- --recursive -R spec"
```
Comment (lukefrake) [inline package.json]: you've got ` -- --recursive ` is that a typo?

Comment (notABluesSinger) [inline package.json]: Yeah, the first `--` is to tell istanbul to pass the subsequent params down to mocha and the second `--` is the mocha param delimiter

File: Gruntfile.js (hunk 1 above)
Comment (notABluesSinger) [inline Gruntfile.js]: The read me already states `npm test` all the things. But good check.

File: package.json (hunk 2 above)
Comment (lukefrake) [inline package.json]: Awesome

File: Gruntfile.js (hunk 1 above)
Comment (lukefrake) [inline Gruntfile.js]: Sorry, couldn't see it when I looked, review fail.

**Pull request merged**